pom.xml

```diff
@@ -212,15 +207,13 @@
     <slf4j.version>2.0.17</slf4j.version>
     <snakeyaml.version>2.0</snakeyaml.version>
     <snappy-java.version>1.1.10.7</snappy-java.version>
-    <sonar.java.binaries>${basedir}/target/classes</sonar.java.binaries>
```

Comment:
we would still need this one for SonarQube analysis: https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/languages/java/